docs/docs/tutorials/using_dask.md

```diff
@@ -31,21 +32,24 @@ Dask consists of 3 main components `client`, `scheduler`, and `workers`.
 
 The `client` interacts with both `scheduler` (sends instructions) and `workers` (collects results).
 
-Check out the [Dask Gateway documentation][dask-gateway] for a full explanation.
+Check out the [Dask documentation][dask-docs] and the [Dask Gateway documentation][dask-gateway] for a full explanation.
 
 </details>
 
-## Step 1 - Setting up Dask Gateway
+## Step 1 - Set up Dask Gateway
 
-Let's start by creating a Jupyter notebook. Select an environment from the `Select kernel` dropdown menu
+Let's start with a fresh Jupyter notebook.
+Select an environment from the `Select kernel` dropdown menu
 (located on the top right of your notebook).
+On a default Nebari deployment, you can select the `filesystem/dask` environment.
 
 :::warning
-Be sure to select an environment which includes `Dask`
+Be sure to select an environment which includes `Dask`, and note that the versions of `dask`, `distributed`, and `dask-gateway` must be the same.
+We recommend the [`nebari-dask` metapackage](https://anaconda.org/conda-forge/nebari-dask) that includes the correct Dask packages and versions.
```

Comment:
```suggestion
We recommend the [`nebari-dask` metapackage](https://anaconda.org/conda-forge/nebari-dask). We publish this metapackage alongside Nebari to easily provide you with the correct Dask packages and versions, just be sure that the `nebari-dask` version matches your Nebari deployment version.
```
Note - we should add a note about bumping the nebari-dask version in the "how to bump the nebari version docs"